deb/openmediavault/var/www/openmediavault/js/omv/module/admin/system/notification/Notification.js

```diff
@@ -175,10 +175,19 @@ Ext.define("OMV.module.admin.system.notification.Settings", {
 					text: _("The default SMTP mail server port, e.g. 25 or 587.")
 				}]
 			},{
-				xtype: "checkbox",
-				name: "tls",
-				fieldLabel: _("Use SSL/TLS secure connection"),
-				checked: false
+				xtype: 'combobox',
```

Comment:
```suggestion
				xtype: "combo",
				fieldLabel: _("Encryption mode"),
				queryMode: "local",
				displayField: "name",
				valueField: "value",
				editable: false,
				name: "tls",
				store: [
					[ "no", _("No encryption") ],
					[ "ssl", _("SSL/TLS") ],
					[ "starttls", _("STARTTLS") ],
					[ "auto", _("Auto") ]
				]
```

Reply:
The [docs](https://docs.sencha.com/extjs/7.0.0/classic/Ext.form.field.ComboBox.html) read like both is possible but I'll change it.